sw/device/silicon_creator/rom/e2e/BUILD

```diff
@@ -446,6 +446,72 @@ test_suite(
     tests = [":e2e_bootstrap_entry_{}".format(lc_state.lower()) for lc_state in structs.to_dict(CONST.LCV)],
 )
 
+otp_json(
+    name = "otp_json_bootstrap_rma",
+    partitions = [
+        otp_partition(
+            name = "CREATOR_SW_CFG",
+            items = {
+                "CREATOR_SW_CFG_RMA_SPIN_EN": "0x739",  # HARDENED_BOOL_TRUE
+                "CREATOR_SW_CFG_RMA_SPIN_CYCLES": "10",
+            },
+        ),
+    ],
+    visibility = ["//visibility:private"],
+)
+
+# This OTP image is tightly coupled with the "rom_bootstrap_rma" testpoint.
+# Despite its name ending with "rma", it actually puts the device into the PROD
+# lifecycle state.
+otp_image(
+    name = "otp_img_bootstrap_rma",
+    src = "//hw/ip/otp_ctrl/data:otp_json_prod",
+    overlays = STD_OTP_OVERLAYS + [
+        ":otp_json_bootstrap_rma",
+    ],
+    visibility = ["//visibility:private"],
+)
+
+bitstream_splice(
+    name = "rom_otp_bootstrap_rma",
+    src = "//hw/bitstream:rom",
+    data = ":otp_img_bootstrap_rma",
+    meminfo = "//hw/bitstream:otp_mmi",
+    tags = [
+        "manual",
```

Comment:
Is this `manual` because of potential flakiness? If so, and if we don't have an issue for similar JTAG based tests, could you please create an issue? Thanks!

Reply:
No, I thought we were marking bitstream splices as manual in case folks don't have Vivado installed, but I see that's not the pattern we're following with nearby bitstreams. Dropping the manual tag :)

Reply:
Great, thanks!